src/test/java/com/conveyal/gtfs/validator/RouteTypeValidatorTest.java

```diff
@@ -0,0 +1,21 @@
+package com.conveyal.gtfs.validator;
+
+import com.google.common.collect.Lists;
+import org.junit.jupiter.api.Test;
+
+import java.util.List;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+class RouteTypeValidatorTest {
+    private static final List<Integer> ROUTE_TYPES = Lists.newArrayList(3, 101);
+    private static final RouteTypeValidator ROUTE_TYPE_VALIDATOR = new RouteTypeValidator(ROUTE_TYPES);
+
+    @Test
+    void validateRouteType() {
+        assertThat(ROUTE_TYPE_VALIDATOR.isRouteTypeValid(3), is(true)); // Bus
```

Comment:
These could be replaced with `assertTrue` and `assertFalse`

Reply:
Addressed in https://github.com/conveyal/gtfs-lib/pull/343/commits/8e9e8c3ee5fe67c9898d21371537fb759a45a21c, thanks!